src/main.c

```diff
@@ -1157,6 +1163,63 @@ static yoml_t *load_config(const char *fn)
     return yoml;
 }
 
+static int filter_config_node(h2o_configurator_context_t *ctx, yoml_t **node)
+{
+    size_t i, j;
+
+    if ((*node)->type != YOML_TYPE_MAPPING)
+        return -1;
+
+    i = (*node)->data.mapping.size;
+    do {
+        --i;
+        if ((*node)->data.mapping.elements[i].key->type == YOML_TYPE_SCALAR &&
+            strcmp((*node)->data.mapping.elements[i].key->data.scalar, "include") == 0) {
+
+            yoml_mapping_element_t src = (*node)->data.mapping.elements[i];
+            if (src.value->type != YOML_TYPE_SCALAR)
+                return -1;
+            yoml_t *included = load_config(src.value->data.scalar);
+            if (included == NULL)
```

Comment:
Can you please adjust the code so that the file and line number that included the file would become part of the error message?